context.md

```diff
@@ -227,6 +227,7 @@ Stacks can be useful to capture historical information about a request, such as
 use Illuminate\Support\Facades\Context;
 use Illuminate\Support\Facades\DB;
 
+// In our service provider...
```

Comment:
Or you could use the `filename` code fence attribute:
````markdown
```php filename=app/Providers/AppServiceProvider.php
use Illuminate\Support\Facades\Context;
use Illuminate\Support\Facades\DB;

DB::listen(function ($event) {
    Context::push('queries', [$event->time, $event->sql]);
});
```
````